### DISCUSSION TITLE
When using Custom Server,page cannot get the return value of getServersideProps method

### DISCUSSION THREAD

**Discussion by lileilei** [vercel/next.js#25092] [Help]
### What version of Next.js are you using?

>= 9.3

### What version of Node.js are you using?

10.2.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

npm start

### Describe the Bug

page method cannot get the return value of getServersideProps method

### Expected Behavior

export default app method, can get the return value of getServersideProps method

```
function Page({data}) {
  console.log(data)  //  **output:  undefined** 
  return(
    <div>{data}</div>
  )
}

export async function getServerSideProps() {
  return { props: { data:[1,2,3] } }
}

export default Page

```

### To Reproduce

use the  "examples/custom-server"

modify  pages/index.js 

```
function Page({data}) {
  console.log(data)  //  output:  undefined
  return(
    <div>{data}</div>
  )
}

export async function getServerSideProps() {
  return { props: { data:[1,2,3] } }
}

export default Page

```

**Comment by timneutkens** [comment]
The reproduction steps work fine:

<img width="437" alt="Screen Shot 2021-05-13 at 13 26 50" src="https://user-images.githubusercontent.com/6324199/118119618-ed508e80-b3ee-11eb-9dc9-5da51a8c8803.png">

**Reply by lileilei** [reply]
Thanks for your answer !
